libs/features/mas/src/merch-card.js

```diff
@@ -363,6 +363,16 @@ export class MerchCard extends LitElement {
     }
 
     async checkReady() {
+        const timeoutPromise = new Promise((resolve) =>
+            setTimeout(() => resolve(false), MERCH_CARD_LOAD_TIMEOUT),
+        );
+        if (this.aemFragment) {
+          const aemLoad = await Promise.race([this.aemFragment.updateComplete, timeoutPromise]);
```

Comment:
i see a pattern here :) , but maybe this could move to aemFragment too (less of a problem to me as it's less supposed to be used by consumers)

Reply:
i could move it, yes. can't reuse same timeout in this case though.